packages/corvid-local-server/test/it/localDocumentChanges.spec.js

```diff
@@ -35,8 +35,11 @@ describe("local changes", () => {
         );
 
         const docItem = sc[itemKey]();
-        let filePath = localSiteBuilder.getLocalFilePath(docItem);
-        let fileContent = localSiteBuilder.getLocalFileContent(docItem);
+        const filePath = localSiteBuilder.getLocalFilePath(docItem, "page");
```

Comment:
asking for "page" is only correct if the itemKey is page/pageWithCode/lightbox/lightboxWithCode